src/task/error.rs

```diff
@@ -6,3 +7,11 @@ use thiserror::Error;
 pub struct MissingTaskError {
     pub task_name: String,
 }
+
+// TODO: We should make this error much better
+#[derive(Debug, Error, Diagnostic)]
+#[error("'{task_name}' is ambiguous")]
```

Comment:
```suggestion
#[error("'{task_name}' is available in environments")]
```
Although ambiguous is the right word this might not be known to every non native English speaker, including myself.

Reply:
Also this error should list the available envs, but I think we can do that later! Just to note.